src/VolleyManagement.Backend/VolleyManagement.UI/Areas/WebAPI/ViewModels/GameReports/StandingsEntryViewModel.cs

```diff
@@ -158,8 +154,23 @@ public static List<StandingsEntryViewModel> SetPositions(List<StandingsEntryView
         public static bool EntriesHaveSamePosition(StandingsEntryViewModel firstEntry, StandingsEntryViewModel secondEntry)
         {
             return firstEntry.Points == secondEntry.Points
-                && firstEntry.SetsRatio == secondEntry.SetsRatio
-                && firstEntry.BallsRatio == secondEntry.BallsRatio;
+                && Equals3DigitPrecision(firstEntry.SetsRatio, secondEntry.SetsRatio)
+                && Equals3DigitPrecision(firstEntry.BallsRatio, secondEntry.BallsRatio);
+        }
+
+        public static bool Equals3DigitPrecision(float? left, float? right)
```

Comment:
Pass tolerance as parameter and use constant

Reply:
ok